code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -333,6 +333,10 @@
 			apply_damage(real_damage, hit_dam_type, hit_zone, armour, sharp=is_sharp, edge=is_edge)
 
 		if(I_DISARM)
+			if(M.stamina <= 5)

```

Comment:
The actual cost subtracted is `M.max_stamina / 6` but the cost to actually _use_ is just. 5? Wat.

Use a var to hold the cost and then reference this.
```dm
var/disarm_cost = M.max_stamina / 6
if (M.stamina <= disarm_cost)
    return
```